ServerCore/Helpers/MailHelper.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Mail;
+using Microsoft.Extensions.Configuration;
+
+/// <summary>
+/// Helper class for sending mail with Mailjet.
+/// </summary>
+public class MailHelper
+{
+    private string PublicSecret;
+    private string PrivateSecret;
+
+    public bool Enabled { get; private set; }
+    private bool IsDev;
+
+    private const string FromAddress = "msph@puzzlehunt.org";
+    private const string ToAddressForBCC = "msph-bcc@puzzlehunt.org";
+
+    public static MailHelper Singleton { get; private set; }
+
+    public static void Initialize(IConfiguration configuration, bool isDev)
+    {
+        Singleton = new MailHelper(configuration, isDev);
+    }
+
+    private MailHelper(IConfiguration configuration, bool isDev)
+    {
+        try
+        {
+            PublicSecret = configuration["Mailjet-Public"];
+            PrivateSecret = configuration["Mailjet-Private"];
+            Enabled = true;
```

Comment:
I'm playing with adding mails for some scenarios and I found this line to be helpful:

            Enabled = PublicSecret != null && PrivateSecret != null;

Reply:
Yeah, I could do it that way. I need the try/catch anyway, since `configuration[]` throws an exception if you haven't set the secrets and I don't know a way to say "get or default".